src/metabase/analytics/prometheus.clj

```diff
@@ -320,6 +346,19 @@
              (catch Exception e
                (log/warn e "Error stopping prometheus web-server")))))))
 
+(defn observe!
+  "Call iapetos.core/observe on the metric in the global registry.
+   Inits registry if it's not been initialized yet."
```

Comment:
I think it would be nice to point out that this is for use with histograms in the docstring. Helpful to Prometheus newbies.

Reply:
good point, added!